docs/defs.md

```diff
@@ -8,9 +8,9 @@ This module provides the definitions for registering a GCC toolchain for C and C
 ## gcc_toolchain
 
 <pre>
-gcc_toolchain(<a href="#gcc_toolchain-name">name</a>, <a href="#gcc_toolchain-binary_prefix">binary_prefix</a>, <a href="#gcc_toolchain-builtin_sysroot_path">builtin_sysroot_path</a>, <a href="#gcc_toolchain-extra_cflags">extra_cflags</a>, <a href="#gcc_toolchain-extra_cxxflags">extra_cxxflags</a>,
-              <a href="#gcc_toolchain-extra_ldflags">extra_ldflags</a>, <a href="#gcc_toolchain-hardcode_sysroot_ld_linux">hardcode_sysroot_ld_linux</a>, <a href="#gcc_toolchain-hardcode_sysroot_rpath">hardcode_sysroot_rpath</a>, <a href="#gcc_toolchain-platform_directory">platform_directory</a>,
-              <a href="#gcc_toolchain-repo_mapping">repo_mapping</a>, <a href="#gcc_toolchain-sha256">sha256</a>, <a href="#gcc_toolchain-strip_prefix">strip_prefix</a>, <a href="#gcc_toolchain-target_arch">target_arch</a>, <a href="#gcc_toolchain-url">url</a>, <a href="#gcc_toolchain-use_builtin_sysroot">use_builtin_sysroot</a>)
+gcc_toolchain(<a href="#gcc_toolchain-name">name</a>, <a href="#gcc_toolchain-bazel_gcc_toolchain_workspace_name">bazel_gcc_toolchain_workspace_name</a>, <a href="#gcc_toolchain-binary_prefix">binary_prefix</a>, <a href="#gcc_toolchain-builtin_sysroot_path">builtin_sysroot_path</a>,
```

Comment:
nit: copy the .gitattributes from the rules-template repo so these deltas aren't shown in code review

Reply:
Done in https://github.com/aspect-build/gcc-toolchain/pull/16/commits/24d6f5782e7d0fbfc728bbc28d7dbe2c068ddd58.